### PR TITLE
Update location for getopt

### PR DESCRIPTION
**Pull request by philoserf**
- `code.google.com/p/getop`t moved to `github.com/pborman/getopt`
- update `gvt` manifest anf `./vendor/`
- incidental gofmt/goimports imports
- minor shell updates addressing a subset of shellcheck warnings

if the pull request is desired upstream we can consider version bump/release.
For your consideration @jszwedko , @meatballhat , and @rafecolton

### CODE REVIEW

File: git_config.go [hunk 1]
```diff
@@ -44,8 +44,8 @@ type GitConfig struct {
 // GetAuthorConfig returns the config source for git author information.
 func GetAuthorConfig(namespace string, setUserConfig bool) (config *GitConfig, err error) {
 	configs := []*GitConfig{
-		&GitConfig{Namespace: namespace, SetUserConfig: setUserConfig, Scope: Local},
-		&GitConfig{Namespace: namespace, SetUserConfig: setUserConfig, Scope: Global},
+		{Namespace: namespace, SetUserConfig: setUserConfig, Scope: Local},
```
Comment (philoserf) [inline git_config.go:47]: `gofmt -s` suggestion for clean reading

File: internal/cmdrunner/execute.go [hunk 2]
```diff
@@ -1,9 +1,8 @@
 package cmdrunner
 
 import (
-	"github.com/git-duet/git-duet/internal/cmd"
-
 	"github.com/git-duet/git-duet"
+	"github.com/git-duet/git-duet/internal/cmd"
```
Comment (philoserf) [inline internal/cmdrunner/execute.go:5]: `goimports` closes up the whitespace and sorts

File: scripts/dist [hunk 3]
```diff
@@ -8,4 +8,4 @@ LDFLAGS="-X main.VersionString=$VERSION -X main.RevisionString=$(git rev-parse -
 
 gox -ldflags "$LDFLAGS" -os 'linux windows darwin' -arch '386 amd64' -output 'dist/{{.OS}}_{{.Arch}}/{{.Dir}}' $(go list ./... | grep -v '/vendor/')
 
-for build in $(find dist/* -type d) ; do tar cvzf $build.tar.gz -C $build . ; done
```
Comment (philoserf) [inline scripts/dist:11]: addressed https://github.com/koalaman/shellcheck/wiki/SC2044

File: scripts/git-duet-rebase.sh [hunk 4]
```diff
@@ -3,18 +3,18 @@
 # courtesy of @pivotaljohn
 
 function usage_and_quit() {
-   echo "Reset author for specified ref through to HEAD to the current duet pair."
-   echo ""
-   echo "Usage:"
-   echo " $0 (base ref)"
-   exit 1
+  echo "Reset author for specified ref through to HEAD to the current duet pair."
```
Comment (philoserf) [inline scripts/git-duet-rebase.sh:6]: just consistent whitespace across scripts

File: git-duet-install-hook/main.go [hunk 5]
```diff
@@ -8,7 +8,7 @@ import (
 	"path"
 	"strings"
 
-	"code.google.com/p/getopt"
```
Comment (philoserf) [inline git-duet-install-hook/main.go:11]: project move announcement https://code.google.com/archive/p/getopt/

**Comment by jszwedko** [comment]
@philoserf awesome, thank you for this and the cleanup.

**Pull request merged**